src/app/record/components/work-stack-group/modals/work-bibtex-modal/work-bibtex-modal.component.ts

```diff
@@ -85,6 +86,10 @@ export class WorkBibtexModalComponent implements OnInit, OnDestroy {
                   .subscribe((data) => {
                     data.forEach((work) => {
                       that.worksFromBibtex.push(work)
+                      if (work.errors.length > 0 && !that.isAnInvalidWork) {
+                        that.isAnInvalidWork = true
```

Comment:
Wondering what happens if the user enters an invalid BibTeX and after that, the user fixes it with a valid BibTeX? 

It seems like we never set the `isAnInvalidWork` back to false 🤔

Reply:
Every time you want to import a new BibTeX file you need to click cancel and open the modal again